my-app/src/HW37/hw-37.tsx

```diff
@@ -0,0 +1,28 @@
+import { Button } from "./buttonBurger"
+import { Title } from "./title";
+import style from "./buttonBurger.module.css"
+import { Tabs } from "./tabs";
+// import styles from "./tabs.module.css"
+
+export const Hw37 = () => (
+    <div className={style.div}>
+        <Title />
+        <div className={style.buttonDiv}>
```

Comment:
I recommend to put styles corresponding to this component put in `hw-37.module.css`